# Do not remove pandas, it is required if beangrow_debug_dir is enabled

diff --git a/src/fava_portfolio_returns/_vendor/beangrow/reports.py b/src/fava_portfolio_returns/_vendor/beangrow/reports.py
index c33b8c6..79cb5f9 100644
--- a/src/fava_portfolio_returns/_vendor/beangrow/reports.py
+++ b/src/fava_portfolio_returns/_vendor/beangrow/reports.py
@@ -19,15 +19,15 @@ from functools import partial
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Tuple
 
-import matplotlib.dates as mdates
-import matplotlib.pyplot as plt
+#import matplotlib.dates as mdates
+#import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 from beancount.core import data, prices
 from beancount.core.amount import Amount
 from beancount.parser import printer
 from dateutil.relativedelta import relativedelta
-from matplotlib.axes import Axes
+#from matplotlib.axes import Axes
 from pandas.plotting import register_matplotlib_converters
 
 from beangrow import investments
@@ -36,7 +36,7 @@ from beangrow.config_pb2 import Config, Group
 from beangrow.investments import AccountData, CashFlow
 from beangrow.returns import Pricer
 
-register_matplotlib_converters()
+#register_matplotlib_converters()
 
 # Basic type aliases.
 Account = str
@@ -374,7 +374,7 @@ def get_accounts_table(account_data: List[AccountData]) -> pd.DataFrame:
 
 
 def set_axis(
-    ax_: Axes,
+    ax_,
     date_min: Date,
     date_max: Date,
 ) -> None:
